easybuild/easyconfigs/g/GCCcore/GCCcore-5.4.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'EB_GCC'
+
+name = 'GCCcore'
+version = '5.4.0'
+
+homepage = 'http://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+mpfr_version = '3.1.4'
+
+source_urls = [
+    'http://ftpmirror.gnu.org/gcc/gcc-%(version)s',  # GCC auto-resolving HTTP mirror
+    'http://ftpmirror.gnu.org/gmp',  # idem for GMP
+    'http://ftpmirror.gnu.org/mpfr',  # idem for MPFR
+    'http://www.multiprecision.org/mpc/download',  # MPC official
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+]
+
+sources = [
+    'gcc-%(version)s.tar.bz2',
+    'gmp-6.1.0.tar.bz2',
+    'mpfr-%s.tar.gz' % mpfr_version,
+    'mpc-1.0.3.tar.gz',
+    'isl-0.15.tar.bz2',
+]
+
+patches = [('mpfr-%s-allpatches-20150603.patch' % mpfr_version, '../mpfr-%s' % mpfr_version)]
```

Comment:
I screwed up the date stamp, this patch is the one I grabbed on June 3rd 2016, so it should be `20160603`; or are you using another way of versioning the patch?